example/interceptor.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
```

Comment:
I would prefer one example.html.  Any new options could be moved into the existing example.